PDSA_System/Server/Controllers/TeamController.cs

```diff
@@ -101,11 +101,11 @@ public async Task<ActionResult<List<Team>>> GetUsersFromTeam(int TeamId)
             var conneString = _configuration.GetValue<string>("ConnectionStrings:DefaultConnection");
             using var conn = new DbHelper(conneString).Connection;
             // denne linjen henter ut teammedlemskap
-            var Brukere = await conn.QueryAsync<TeamMedlemskap>("SELECT * FROM TeamMedlemskap WHERE TeamId = @id",
+            var brukere = await conn.QueryAsync<TeamMedlemskap[]>("SELECT * FROM TeamMedlemskap WHERE TeamId = @id",
```

Comment:
SQL

SELECT T.AnsattNr, B.Fornavn, B.Etternavn FROM TeamMedlemskap AS T, Bruker AS B WHERE T.AnsattNr = B.AnsattNr AND T.TeamId = 2;